### PR TITLE
Fix GB refocus issue

### PR DESCRIPTION
**Pull request by msmanoj**
## Description
- Fixes the issue that breaks the nav bar when the bookmark link is clicked twice.

Also fixes the following `stylelint` warning that appears on save:
```
WARNING in [stylelint] 
src/header/launchbar/Launchbar.scss
 51:1  ⚠  Expected selector ".launchbarButton" to come      no-descending-specificity
          before selector ".category:first-child                                     
          .launchbarButton"                                                          
 51:1  ⚠  Expected selector ".launchbarButton" to come      no-descending-specificity
          before selector ".category:first-child                                     
          .launchbarButton:first-child"

```

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1681

## Deployment URL(s)
http://fix-gb-gene-refocus.review.ensembl.org

## Views affected
Genome browser -> bookmarks

### CODE REVIEW

File: src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx [hunk 1]
```diff
@@ -55,16 +57,22 @@ export const PreviouslyViewedLinks = () => {
           focus: buildFocusIdForUrl(previouslyViewedObject.object_id)
         });
 
+        const isFocusCurrentlyActive =
+          focusObjectId === previouslyViewedObject.object_id;
+
+        const onClick = isFocusCurrentlyActive
+          ? (event: React.MouseEvent<HTMLAnchorElement>) => {
+              changeFocusObject(focusObjectId);
+              event.preventDefault();
```
Comment (imransl) [inline src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx:66]: I've usually seen `event.preventDefault()` put in the top of the function. But think this should be okay too.

File: src/content/app/genome-browser/components/zmenu/ZmenuAppLinks.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ import { ToggleButton as ToolboxToggleButton } from 'src/shared/components/toolb
 import ViewInApp, { UrlObj } from 'src/shared/components/view-in-app/ViewInApp';
 
 import styles from './Zmenu.scss';
+import useGenomeBrowser from '../../hooks/useGenomeBrowser';
```
Comment (imransl) [inline src/content/app/genome-browser/components/zmenu/ZmenuAppLinks.tsx:28]: Don't like to be the import statements pedant, but just noticed this import is in the wrong place.

File: src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx [hunk 3]
```diff
@@ -55,16 +57,22 @@ export const PreviouslyViewedLinks = () => {
...
           <div
             key={previouslyViewedObject.object_id}
             className={styles.linkHolder}
           >
-            <Link
-              replace
-              to={path}
-              onClick={() => onLinkClick(previouslyViewedObject.type, index)}
-            >
+            <Link replace to={path} onClick={onClick}>
```
Comment (msmanoj) [inline src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx:75]: This is correct here. I thought this is ViewInApp component where `Link` will not be needed.

File: src/content/app/genome-browser/components/zmenu/ZmenuAppLinks.tsx (hunk 2 above)
Comment (msmanoj) [reply src/content/app/genome-browser/components/zmenu/ZmenuAppLinks.tsx]: Updated it

File: src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx (hunk 1 above)
Comment (azangru) [reply src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/BookmarksModal.tsx:66]: I'd also put it in the beginning of the function, out of habit. Don't think it really matters one way or the other though.

**Review by azangru — APPROVED** [review]

**Review by imransl — APPROVED** [review]

**Pull request merged**